### PR TITLE
Fix/typos

### PR DESCRIPTION
**Pull request by 0x3bfc**
https://github.com/aurora-is-near/doc.aurora.dev/runs/3204773851

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ nav:
     - Getting Started:
       - MetaMask: develop/start/metamask.md
       - Truffle: develop/start/truffle.md
-      - Hardhat: develop/start/hardhat.md
+      - HardHat: develop/start/hardhat.md
```
Comment (birchmd) [inline mkdocs.yml:39]: I thought we agreed "Hardhat" was the correct spelling

Comment (birchmd) [reply mkdocs.yml:39]: E.g. from [their website](https://hardhat.org/) "Hardhat is next-generation Ethereum tooling"

Comment (0x3bfc) [reply mkdocs.yml:39]: Ops ... that commit was from an old branch. will fix it.

**Review by birchmd — APPROVED** [review]
Thanks for fixing these!

**Pull request merged**